hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -297,7 +298,7 @@ public static void main(final String... args) throws Exception {
         final HashedReservedSignedState reservedState = loadInitialState(
                 platformConfig,
                 recycleBin,
-                version,
+                version.getPbjSemanticVersion(),
```

Comment:
@timfn-hg This change was very subtle and caused some issues not detecting upgrade, restart correctly from previous experiences 🙂 . Could you please test upgrade, restart works fine with the change.